tests/test_manager.py

```diff
@@ -409,6 +409,9 @@ def test_two_miners_same_submission_1(self):
         conn1.send_result = MagicMock(return_value=None)
         self.manager.backend.push_tx_or_block = MagicMock(return_value=asyncio.Future())
         conn1.method_submit(params=params, msgid=None)
+
+        self._run_all_pending_events()
```

Comment:
`self.manager.backend.push_tx_or_block` is now called from inside a async method in the manager. So I needed to add this here (and below) to make sure this async method would have been already called before the assertions.